src/components/Scroller/docs.md

```diff
@@ -0,0 +1,64 @@
+## Scroller
+
+The `Scroller` Component arranges its children in a flex row and allows for horizontal scrolling if the content size exeeds the container size. It is the basis of the `<Tab type='scroll' />`components.
+
+Supported Props
+
+- `children` (ReactNode, required):
+- `center`? (boolean, false): centers the child nodes when container with exeeds content width.
+- `activeScrollItemIndex`? (number, 0): The index of the active scroll item
+- `hideArrows`? (boolean, false): Hides the arrows
+- `arrowSize`? (number, 28): Arrow Size
+- `breakoutWidth`? (number, 0): If the Scroller is in a container with negative horizontal margins, pass those as breakoutWidth to allow for proper padding in full width layouts
```

Comment:
I think the issue is that I accidentally made it seem like breakout is dependent on arrows being present. Which it doesn't really, it just relies on the same state as arrows, because "right" just measures if the scroll box is larger than its container which is what is used to use the breakout values or not. I don't want to limit the usability of this component to articles, which is why I'm reluctant to only support a const.